casr/src/bin/casr-libfuzzer.rs

```diff
@@ -170,9 +191,28 @@ fn main() -> Result<()> {
             casr_cmd.args(argv.clone());
             casr_cmd.arg(crash);
             debug!("{:?}", casr_cmd);
-            let casr_output = casr_cmd
-                .output()
-                .with_context(|| format!("Couldn't launch {casr_cmd:?}"))?;
+
+            // Get output
+            let casr_output =
+            // If timeout is specified, spawn and check timeout
+            // Else get output
+            if timeout != 0 {
+                let mut child = casr_cmd
+                    .spawn()
+                    .with_context(|| "Failed to start command: {casr_cmd:?}")?;
+                if child
+                    .wait_timeout(Duration::from_secs(timeout))
+                    .unwrap()
+                    .is_none()
+                {
+                    child.kill()?;
+                    warn!("Timeout: {:?}", casr_cmd);
+                }
+                child.wait_with_output()?
+            } else {
+                casr_cmd.output().with_context(|| format!("Couldn't launch {casr_cmd:?}"))?
```

Comment:
we should support timeout in casr-san, etc. and call it with -t here

Reply:
Done (except casr-gdb so far)

Reply:
Done for casr-gdb too